py3status/modules/file_status.py

```diff
@@ -77,13 +79,17 @@ class Meta:
         }
 
     def post_config_hook(self):
+        self.placeholders = set(self.py3.get_placeholders_list(self.format_path))
+
         if self.path:
             # backward compatibility, str to list
             if not isinstance(self.path, list):
                 self.path = [self.path]
             # expand user paths
             self.path = list(map(expanduser, self.path))
 
+            self.format_path_separator = self.py3.safe_format(self.format_path_separator)
```

Comment:
Smart move. I did it before. However, the problem is that this won't work with thresholds, eg `'\?color=paths ,` to follow number of `paths` or such. It's not likely to happen here, but could happen in other modules so we keep them in the main.

Reply:
Ok, i reput it in the main so